internal/pkg/show/config/plugins/jenkins.yaml

```diff
@@ -22,8 +19,6 @@ tools:
       chart:
         # name of the chart
         chart_name: jenkins/jenkins
-        # release name of the chart
-        release_name: dev
```

Comment:
Is it on purpose or by mistake? @daniel-hutao

Reply:
purpose